benchmarks/microbenchmarks/_result.py

```diff
@@ -18,11 +31,17 @@ class ResultType(BenchmarkItem):
     name = 'get_result'
```

Comment:
Maybe change to `MeasurerType`?

Reply:
Awesome! How about renaming it as `MetricType`
it is possible to extend the observed  metrics beyond the `elapsed time` (kernel and e2e)